test/unit/pipeline/FetchUnitTest.cc

```diff
@@ -163,7 +172,8 @@ TEST_F(PipelineFetchUnitTest, fetchAligned) {
       .WillByDefault(DoAll(SetArgReferee<3>(mOp), Return(4)));
```

Comment:
Similar commnets apply here regarding predecode return value and EXPECT_CALL times expected